src/PictureCard/index.js

```diff
@@ -0,0 +1,276 @@
+  transition: all ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+  bottom: 0;
+`;
+
+StyledPictureCardContent.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledPictureCard = styled(({ height, href, theme, external, shadows, ...props }) => {
+  const Component = href ? "a" : "div";
+  return <Component {...props}>{props.children}</Component>;
+})`
+  height: ${({ height }) => (height ? `${height}` : "100%")};
+  width: ${({ width }) => (width ? `${width}` : `100%`)};
+  display: flex;
+  justify-content: space-between;
+  align-content: flex-end;
+  flex-direction: column;
+  position: relative;
+  box-sizing: border-box;
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  overflow: hidden;
+  cursor: pointer;
+  box-shadow: ${({ theme, shadows }) => shadows && theme.orbit.boxShadowAction};
+
+  ${StyledLazyImage} {
+    transition: transform ${({ theme }) => theme.orbit.durationSlow} ease-in-out;
+  }
+
+  ${({ isPlain, theme, shadows }) =>
+    !isPlain &&
+    css`
+      &:hover,
+      &:focus {
+        outline: none;
+        box-shadow: ${shadows && theme.orbit.boxShadowActionActive};
+
+        ${StyledOverlayHover} {
+          opacity: 1;
+        }
+
+        ${StyledLazyImage} {
+          transform: scale(1.05);
+        }
+
+        ${StyledPictureCardContent} {
+          bottom: ${({ contentHeight }) => contentHeight && `${contentHeight}px`};
+        }
+
+        ${Shown} {
+          opacity: 1;
+          bottom: 0;
+        }
+      }
+    `}
+`;
+
+StyledPictureCard.defaultProps = {
+  theme: defaultTheme,
+};
+
+const PictureCard = ({
+  onClick,
+  image,
+  title,
+  tabIndex,
+  subTitle,
+  href,
+  children,
+  external,
+  actions,
+  label,
+  height = "300px",
+  width,
+  dataTest,
+}: Props) => {
+  const ref: { current: any | HTMLDivElement } = React.useRef(null);
+  const [contentHeight, setContentHeight] = React.useState(0);
+
+  React.useEffect(() => {
+    if (ref.current) {
+      setContentHeight(ref.current.clientHeight);
+    }
+  }, []);
+
+  const cardID = React.useMemo(() => randomID("pictureCardID"), []);
+
+  const handleKeyDown = (ev: SyntheticKeyboardEvent<HTMLElement>) => {
+    if (onClick) {
+      if (ev.keyCode === KEY_CODE_MAP.ENTER) {
+        onClick();
+      } else if (ev.keyCode === KEY_CODE_MAP.SPACE) {
+        ev.preventDefault();
+        onClick();
+      }
+    }
+  };
+
+  const { name, original, placeholder, code } = image;
+  const isPlain = !(title || subTitle || children || actions);
+
+  return (
+    <StyledPictureCard
+      data-test={dataTest}
+      onClick={onClick}
+      onKeyDown={handleKeyDown}
+      height={parseInt(height, 10) >= SMALLEST_HEIGHT ? height : SMALLEST_HEIGHT}
+      width={width}
+      href={href}
+      external={external}
+      shadows={onClick || href}
+      tabIndex={href ? tabIndex : 0}
+      role="link"
+      contentHeight={contentHeight}
+      isPlain={isPlain}
+      aria-labelledby={cardID}
```

Comment:
The `cardID` should be used also somewhere else for `id` on div, preferably on `PictureCardContent` element.